config.js

```diff
@@ -3,28 +3,28 @@ var APP_NAME = 'Mapeo'
 var APP_TEAM = 'Digital Democracy'
 var APP_VERSION = require('./src/build-config').version
 
-var isElectron = typeof process.type === 'string'
-// Is `true` when running from Node
-var isDev = isElectron ? require('electron-is-dev') : true
+function getResourcesDir (isDev) {
+  // If running from Node, process.type is not defined
+  const isElectron = typeof process.type === 'string'
+  if (!isElectron || isDev) return path.join(__dirname, 'temp-resources')
+  return process.resourcesPath
+}
 
-// Sorry about this! In production the default config is shipped in the app
-// resources folder, but for development we need to copy them into a temporary
-// "resources" folder.
-var RESOURCES_DIR = isDev
-  ? path.join(__dirname, 'temp-resources')
-  : process.resourcesPath
+function getDefaultConfigDir (isDev) {
+  // This is super confusing... due to hard-coded paths in @mapeo/settings
+  // TODO: Clean all of this up in mapeo-server and @mapeo/settings
+  return path.join(getResourcesDir(isDev), 'presets')
+}
 
 module.exports = {
   APP_NAME,
   APP_TEAM,
   APP_VERSION,
-  RESOURCES_DIR,
-  // This is super confusing... due to hard-coded paths in @mapeo/settings
-  // TODO: Clean all of this up in mapeo-server and @mapeo/settings
-  DEFAULT_CONFIG_DIR: path.join(RESOURCES_DIR, 'presets'),
   MAPBOX_ACCESS_TOKEN:
     'pk.eyJ1IjoiZ21hY2xlbm5hbiIsImEiOiJSaWVtd2lRIn0.ASYMZE2HhwkAw4Vt7SavEg',
   GITHUB_URL: 'https://github.com/digidem/mapeo-desktop',
   GITHUB_URL_RAW:
-    'https://raw.githubusercontent.com/digidem/mapeo-desktop/master'
+    'https://raw.githubusercontent.com/digidem/mapeo-desktop/master',
+  getResourcesDir,
+  getDefaultConfigDir
```

Comment:
replaced the statically defined constants with these functions because this file is run in various contexts that are sensitive to the environment e.g. main process, background renderer, primary window renderer